### PR TITLE
chore(sns): Remove neuron_parameters from ClaimSwapNeuronsRequest on SNS Governance's side

### PR DESCRIPTION
**Pull request by anchpop**
The field is deprecated and no longer supported. This change is safe because neuron_parameters was already not supported by `Governance`, this just removes the field entirely from the request it expects. 

[← Previous PR](https://github.com/dfinity/ic/pull/1295)

### CODE REVIEW

**Review by max-dfinity — APPROVED** [review]

File: Cargo.lock [hunk 1]
```diff
@@ -16874,9 +16874,9 @@ dependencies = [
 
 [[package]]
 name = "quinn"
-version = "0.11.4"
+version = "0.11.5"
```
Comment (rumenov) [inline Cargo.lock:16877]: this seems like unrelated change
